lib/i18n-ai/clients/anthropic_client.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "anthropic"
+require_relative "base_client"
+
+module I18nAi
+  module Clients
+    class AnthropicClient < BaseClient
+      def initialize
+        super()
```

Comment:
do you mind removing `()` since it's an implicit function call without the `()` and more ruby-ish. thank you.